.github/workflows/release-next.yaml

```diff
@@ -88,19 +87,10 @@ jobs:
       - package
     runs-on: ubuntu-latest
     steps:
-      - name: Cache bin path
-        id: cache
-        uses: actions/cache@v2
-        with:
-          path: ~/bin
-          key: ${{ runner.os }}-${{ needs.setup.outputs.HELM_VERSION }}-${{ needs.setup.outputs.YQ_VERSION }}
       - name: Checkout
         uses: actions/checkout@v2
-        if: steps.cache.outputs.cache-hit != 'true'
-      - run: scripts/install-helm.sh $HELM_VERSION
-        if: steps.cache.outputs.cache-hit != 'true'
-      - run: scripts/install-yq.sh $YQ_VERSION
-        if: steps.cache.outputs.cache-hit != 'true'
+      - run: scripts/install-helm.sh ${{ needs.setup.outputs.HELM_VERSION }}
```

Comment:
Was there a reason to remove the caches?

Reply:
Yes, I explained it in the PR description:

>A problem was spotted with the cache that's being used for the helm and yq binaries: since the install uses a script, we need to perform a checkout in case we get a cache miss. It wasn't the case in the next workflow, which led to some failures.
The install being very fast (checkout + install helm + install yq takes about 3 seconds), I removed the cache everywhere in favor of performing the installs each time. This will also help us spot bugs that could be introduced in the install scripts.

This is all debatable of course, but the cache didn't seem to bring any significant improvement and was hiding some errors in the workflows.